### PR TITLE
Code Review for DFPK Pi ZeroW

### PR DESCRIPTION
**Pull request by bertokhoury**
The page can be accessed at 192.168.42.1/cgi-bin/dfpk-wifi

Later this page will become a captive portal by acting as a FAS for
nodogsplash

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -67,7 +67,7 @@ ENV PACKAGES=git-http
...
...crypto-misc kmod-crypto-user kmod-fs-ext4 kmod-fs-f2fs kmod-gpio-button-hotplug kmod-i2c-algo-bit kmod-i2c-bcm2708 kmod-i2c-core kmod-ip6tables kmod-ipt-conntrack kmod-ipt-core kmod-ipt-nat kmod-ledtrig-netdev kmod-lib-crc-ccitt kmod-mac80211 kmod-nf-conntrack kmod-nf-conntrack6 kmod-nf-ipt kmod-nf-ipt6 kmod-nf-nat kmod-nf-nathelper kmod-nls-base kmod-ppp kmod-pppoe kmod-pppox kmod-slhc kmod-usb-acm kmod-usb-core kmod-usb2 kmod-usb-dwc2 kmod-usb-gadget-eth kmod-usb-net kmod-usb-net-cdc-ether kmod-usb-net-rndis less libblobmsg-json libgmp libip4tc libip6tc libiwinfo libiwinfo-lua libjson-c libjson-script liblua libnl-tiny libubox libubus libubus-lua libuci libuci-lua libustream-openssl libxtables libyaml lua luci luci-app-firewall luci-app-splash luci-base luci-lib-ip luci-lib-nixio luci-mod-admin-full luci-proto-ipv6 luci-proto-ppp luci-theme-material luci-theme-openwrt make mtd netifd obfsproxy opkg ppp ppp-mod-pppoe procd python-crypto python-dev python-pip python-pyptlib python-smbus python-yaml resize2fs rpcd sfdisk swconfig tcpdump tor tor-geoip twisted uboot-envtools ubox ubus ubusd uci uhttpd uhttpd-mod-ubus usign vim wpad-mini zope-interface nodogsplash openssh-sftp-server"
```
Comment (ianblenke) [inline Dockerfile]: I get the `nodogsplash`, but do we need the `openssh-sftp-server` for anything, or is that just a debugging addition?

**Review by ianblenke — CHANGES_REQUESTED** [review]
It looks like the initial wifi configuration stuff as a captive portal is here. I'm guessing that the tor configuration stuff is still under development.

File: Dockerfile (hunk 1 above)
Comment (bertokhoury) [reply Dockerfile]: Yeah that was just to be able to shh into it.

Comment (ianblenke) [reply Dockerfile]: Any more improvements to this?

Comment (MasonAsh) [reply Dockerfile]: I've added a page to fetch and apply the tor config from the REST api. The next steps are to allow the config to be fetched from an onion address and then to setup the `dfpk-wifi` page as a forward authenticating service for `nodogsplash`.

Comment (ianblenke) [reply Dockerfile]: I see some changes to use tor as a socks5 proxy... is this ready for me to try yet?

Comment (bertokhoury) [reply Dockerfile]: Yes, socks5 proxy is good to go.

**Comment by ianblenke** [comment]
Merging and taking it from here. Merry Christmas!

**Pull request merged**